python/features.bzl

```diff
@@ -13,6 +13,9 @@
 # limitations under the License.
 """Allows detecting of rules_python features that aren't easily detected."""
 
+_VERSION_PRIVATE = "$Format:%(describe:tags=true)$"
```

Comment:
Could you please point to `git archive` manual in here. It is very much unclear how this gets populated in reality. Pointing to `.gitattributes` in the comment would be also good.

Reply:
Done